recipes/r-hdrcde/meta.yaml

```diff
@@ -1,94 +1,49 @@
+{% set version = '3.1' %}
```

Comment:
Not the name, only the version as it is proposed by the newest conda-build. Also having the package name as a variable is overkill, because it never changes. The point of having the version as a variable is that you can simply change the version in one place and the rest of the recipe adjusts automatically. You don't need to do that for recipes you submitted before. Just the stuff that is new from now on.